elbs/virginia/snippets-virginia.tfvars

```diff
@@ -0,0 +1,6 @@
+snippets_elb_name = "snippets"
+snippets_subnets = "subnet-43125f6e,subnet-a699aaef,subnet-b6ceb6ed"
+snippets_instances = "i-007cad57f0c5b3157,i-01fd2b1570e510dfe,i-04c56e5c677568159,i-0981f33bdf95c3c1f,i-0f71cd35d3bb09512"
```

Comment:
ah, yes, I missed that and it's critical.

We might not need kops/tf import at all, as the ASG name is just `nodes.virginia.moz.works`. Then, just a call to  `aws autoscaling attach-load-balancers ...`. I'd prefer not to tough the ASG's with TF if possible.